recipes/opengl/all/conanfile.py

```diff
@@ -58,7 +58,7 @@ def system_requirements(self):
             else:
                 self.output.warn("Don't know how to install OpenGL for your distro.")
         elif tools.os_info.is_freebsd and self.settings.os == "FreeBSD":
-            packages = ["mesa-libs"]
+            packages = ["libglvnd"]
```

Comment:
Is this available in all versions of the last few years?

Reply:
my source is https://pkgs.org/search/?q=libglvnd, it shows both FreeBSD 12 and 13 have it, but they are the only versions available on this website :/

Reply:
Seems like FreeBSD 12 was published in December 2018, but the package was added 2020-05-27 according to https://cgit.freebsd.org/ports/log/graphics/libglvnd

Maybe they backport packages to still supported versions? Anyway, we can still improve this later if there is demand for it I guess